test/run-spec-wasm2c.py

```diff
@@ -341,10 +341,13 @@ def Compile(cc, c_filename, out_dir, *args):
     if IS_WINDOWS:
         args += ['/nologo', '/MDd', '/c', c_filename, '/Fo' + o_filename]
     else:
-        args += ['-c', c_filename, '-o', o_filename,
+        args += ['-c', c_filename, '-o', o_filename, '-O2',
```

Comment:
Lets add some comment here maybe?  And perhaps move reference these flags in the wasm2c README file?

What about `-O3`, does that break anything?

Reply:
Just checked and it still works iwth -O3 (at least with gcc on Linux).

Reply:
Added a discussion of the flags to the wasm2c README.